tests/README.md

```diff
@@ -119,17 +119,17 @@ Properties:
 
 Local Test Process:
 ====
-* -Plocal enables failsafe-maven-plugin processing of @Local annotations
+* by default failsafe-maven-plugin processes @LocalOnly annotations
 * com.spotify:docker-maven-plugin builds target container based on value of *jetty.test.image*
 * io.fabric8:docker-maven-plugin starts the target container in pre-integration-test phase
   * random local port mapped to 8080 of container and available to test case as system property *app.deploy.port*
 * failsafe-maven-plugin runs in integration-test phase
-* io.fabric8:docker-maven-plugin stops the target container in  post-integration-test phase
+* io.fabric8:docker-maven-plugin stops the target container in post-integration-test phase
 
 Remote Test Process:
 ====
 
-* -Premote enables failsafe-maven-plugin processing of @Remote annotations
+* -Dtest.remote enables failsafe-maven-plugin processing of @RemoteOnly annotations
```

Comment:
My comment was more about `-Dtest.remote`. Was it a typo and you wanted to write `-Ptest.remote` because `test.remote` is a profile, not a property.